### PR TITLE
[test] Revisit LAMMPS CPU large performance

### PR DESCRIPTION
**Pull request by teojgo**

### CODE REVIEW

**Review by lucamar — APPROVED** [review]
Thanks @teojgo, unfortunately the performance has degraded a lot in the past week: I assume this is due to the multicore nodes, recently busier than usual

File: cscs-checks/apps/lammps/lammps_check.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ class LAMMPSCPUCheck(LAMMPSCheck):
             'pilatus:mc': {'perf': (5000, -0.10, None, 'timesteps/s')}
         },
         'large': {
-            'daint:mc': {'perf': (5360, -0.30, None, 'timesteps/s')},
+            'daint:mc': {'perf': (3477, -0.30, None, 'timesteps/s')},
```
Comment (vkarak) [inline cscs-checks/apps/lammps/lammps_check.py:94]: According to my latest records, this is the value I get... 

```suggestion
            'daint:mc': {'perf': (2875, -0.65, None, 'timesteps/s')},
```

Comment (lucamar) [reply cscs-checks/apps/lammps/lammps_check.py:94]: The performance might increase at the beginning of the new quarter as of tomorrow: today I could get `4326 timesteps/s` for the large check (by the way, using `--strict` did not raise a failure).

**Comment by teojgo** [comment]
Closing since the performance is back to the expected one.

**Pull request closed without merging**